src/main/java/uk/gov/dvsa/logging/LoggingExecutor.java

```diff
@@ -32,4 +33,9 @@ private void logEvent(EventType event, String message, long start) {
         logger.info(message, duration);
         LogContextWrapper.cleanupDuration();
     }
+
+    private void logEvent(EventType event) {
+        LogContextWrapper.setEvent(event);
+        logger.info(event.getDescription());
```

Comment:
For lines 18 & 39, no text stating what is being logged here?

Reply:
The text is held within the event object's description attribute which is basically the strings in this enum... 
![image](https://user-images.githubusercontent.com/92087051/199031863-f7dda313-07ee-4866-92f8-0d2fd7b13f23.png)